test/src/test/java/org/corfudb/integration/LogReplicationAbstractIT.java

```diff
@@ -160,9 +173,14 @@ public void testEndToEndSnapshotAndLogEntrySyncUFO(int totalNumMaps) throws Exce
         final int totalStandbyStatusUpdates = 2;
 
         try {
-            log.debug("Setup active and standby Corfu's");
+            log.info(">> Setup active and standby Corfu's");
```

Comment:
nit - change back to debug?

Reply:
It's fine actually its better in info, cause when you're debugging, 'DEBUG' levels in the tests brings too much level of detail and these statements are normally meant just to guide through which step it's getting stuck...